lib/lita/handlers/jira_issues.rb

```diff
@@ -80,6 +82,26 @@ def priority(data)
       def issue_link(key)
         "\n#{config.url}/browse/#{key}"
       end
+
+      def silenced?(key)
+
+        if config.issue_ttl == 0
+          log.debug("JiraIssues: issue_ttl is set to 0, will post every matched issue to chat")
+          return false
+        end
+
+        current_ttl = redis.ttl(key).to_i
```

Comment:
I'm wondering if `to_i` is the right conversion to use here. `to_i` is very lenient, whereas `Integer(redis.ttl(key))` is stricter in the conversion...